src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -188,19 +191,7 @@ internal override CommonAnonymousTypeManager CommonAnonymousTypeManager
         /// <summary>
         /// True if we should enable nullable semantic analysis in this compilation.
         /// </summary>
-        internal bool NullableSemanticAnalysisEnabled
-        {
-            get
-            {
-                var nullableAnalysisFlag = Feature("run-nullable-analysis");
-                if (nullableAnalysisFlag == "false")
-                {
-                    return false;
-                }
-
-                return ShouldRunNullableWalker || nullableAnalysisFlag == "true";
-            }
-        }
+        internal bool NullableSemanticAnalysisEnabled => ShouldRunNullableAnalysis;
```

Comment:
Why leave this in if it just returns `ShouldRunNullableAnalysis`?

Reply:
Agree, think we should rename here because the `Should` here is getting a bit ambiguous in the scope of whether we can or can't run this.